lib/scholars_archive/validators/graduation_year_validator.rb

```diff
@@ -1,7 +1,7 @@
 module ScholarsArchive::Validators
   class GraduationYearValidator < ActiveModel::Validator
     def validate(record)
-      return if record.graduation_year.empty? || valid_value(record)
+      return if record.graduation_year.present? || valid_value(record)
```

Comment:
I think this should be `!record.graduation_year.present?` instead of `record.graduation_year.present?`.

Reply:
I think you are right. I went off of what was in this ticket
https://github.com/osulp/Scholars-Archive/issues/1255